requirements.txt

```diff
@@ -1,6 +1,6 @@
 numpy==1.11.0
 networkx==1.11
--e git+https://github.com/pogodevorg/pgoapi.git/@3a02e7416f6924b1bbcbcdde60c10bd247ba8e11#egg=pgoapi
+-e git+https://github.com/pogodevorg/pgoapi.git#egg=pgoapi
```

Comment:
Thank you guys, I'll make the change, I knew i had to be missing something in case the repository gets updated